src/main/java/com/mlaf/hu/decisionagent/implementationexample/ImplementationExampleDecisionAgent.java

```diff
@@ -0,0 +1,30 @@
+package com.mlaf.hu.decisionagent.implementationexample;
+
+import com.mlaf.hu.decisionagent.DecisionAgent;
+import com.mlaf.hu.helpers.JadeServices;
+import com.mlaf.hu.models.Plan;
+import jade.core.AID;
+import jade.lang.acl.ACLMessage;
+
+public class ImplementationExampleDecisionAgent extends DecisionAgent {
+
+    @Override
+    public void unregisterSensorAgent(AID sensoragent) {
+
+    }
+
+    @Override
+    public void storeReading(double value) {
+
+    }
+
+    @Override
+    public void executePlan(Plan plan) {
```

Comment:
Dit is wat verwarrend. Er staat toch al in het plan wat er moet gebeuren wanneer deze getriggerd wordt? bijvoorbeeld: stuur een e-mail. Gebeurd dat dan óók en is dit aanvullend? of zijn de consequenties uit een plan fictief?

Reply:
Agreed. Die moet in de abstracte Decision Agent klasse afgehandeld worden. Ga ik verplaatsen. Had het eerst wel op die manier.